src/chrome/internal/exceptions/pauseOnExceptionRequestHandlers.ts

```diff
@@ -44,12 +44,14 @@ export class PauseOnExceptionRequestHandlers implements ICommandHandlerDeclarer
         }
     }
 
-    private toPauseOnPromiseRejectionsStrategy(exceptionFilters: string[]): IPauseOnPromiseRejectionsStrategy {
-        if (exceptionFilters.indexOf('promise_reject') >= 0) {
-            return new PauseOnAllRejections();
-        } else {
-            return new DoNotPauseOnAnyRejections();
-        }
+    private toPauseOnPromiseRejectionsStrategy(_exceptionFilters: string[]): IPauseOnPromiseRejectionsStrategy {
+        return new PauseOnAllRejections();
```

Comment:
Does this mean we will pause on all rejections regardless of whether the feature is turned on or not?

Reply:
At the moment we pause on all rejections if the "Uncaught exception" option is turned on. I tested a "caught" rejection, and we also pause there, which seems a bug. I tested this on V1 and Chrome Dev Tools and they both pause on "caught" rejections too (We get a Debugger.pause message which says: "uncaught": true, so this is probably an issue with Chrome/V8)